projects/packages/stats/src/class-xmlrpc.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * The Stats XMLRPC class
+ *
+ * @package automattic/jetpack-stats
+ */
+
+namespace Automattic\Jetpack\Stats;
+
+use Automattic\Jetpack\Constants;
+
+/**
+ * Stats XMLRPC.
+ *
+ * Adds additional methods to the WordPress XML-RPC API for handling Stats specific features.
+ *
+ * @since $$next-version$$
+ */
+class XMLRPC {
```

Comment:
Does the class name explain its purpose well? For me, it doesn't :) Something like `XMLRPCProvider` might be more self explanatory.

Reply:
I chose this name mostly to be consistent with [similar implementations in other packages](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/videopress/src/class-xmlrpc.php).
I'm fine changing it though as I agree with you :)